xml_inscriptions/transcribed/KY.Lou.SAM.L.1929.17.417.xml

```diff
@@ -108,12 +108,18 @@
       
         <body>
         <div type="edition">
-         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
+         <ab>
+            <lb n="1"/> <expan><abbr>D</abbr><ex>is</ex></expan> <expan><abbr>M</abbr><ex>anibus</ex></expan>.
+            <lb n="2"/> <persName><name type="praenomen" key="Lucius"><expan><abbr>L</abbr><ex>ucio</ex></expan></name>
+                                  <name type="gentilicium" key="Publilius">Pu<choice><sic>p</sic><corr>b</corr></choice>lilio</name>
+                                  <name type="cognomen" key="Serapionis">Sera
+            <lb n="3" break="no"/> pioni</name></persName>.
```

Comment:
Can't have the space between the `<lb>` element and the rest of the text of the name; it'll then add the space in the text itself.